### PR TITLE
Add Entry: C Structures

### PR DESCRIPTION
**Pull request by KTom101**
This is a continuation of [PR #822](https://github.com/Codecademy/docs/pull/822). Fork for previous PR was deleted by mistake.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c/concepts/structures/structures.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+## Structure Variables
+
+[Variables](https://www.codecademy.com/resources/docs/c/variables) can be created in order to work with the `Patient` structure above. The following example declares and initializes a structure variable:
+
+```c
+#include <stdio.h>
+#include <string.h>
+
+struct Patient{
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:56]:
```suggestion
struct Patient {
```

File: content/c/concepts/structures/structures.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  char name[30];
+  int age;
+  int weight;
+  int heightInInches;
+};
+
+int main() {
+
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:64]:
```suggestion
```

File: content/c/concepts/structures/structures.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  // Variable initialization at declaration
+  struct Patient patientA = {"Douglas Franklin", 62, 280, 74};
+  return 0;
+
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:68]:
```suggestion
```

File: content/c/concepts/structures/structures.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+## Accessing Members
+
+Members can be accessed using the member operator `.`:
+
+```c
+#include <stdio.h>
+#include <string.h>
+
+struct Patient{
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:80]:
```suggestion
struct Patient {
```

File: content/c/concepts/structures/structures.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+```c
+#include <stdio.h>
+#include <string.h>
+
+struct Hospital {
+    char department[30];
+    int roomNum;
+};
+
+struct Patient{
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:120]:
```suggestion
struct Patient {
```

File: content/c/concepts/structures/structures.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+};
+
+int main() {
+  // Variable initialization at declaration
+  struct Patient patientA = {"Douglas Franklin", 62, 280, 74};
+
+  // Accessing members
+  printf("Name: %s \nAge: %d \nWeight: %d \nHeight: %d\n",
+        patientA.name, patientA.age, patientA.weight, patientA.heightInInches);
+  return 0;
```
Comment (yangc95) [inline content/c/concepts/structures/structures.md:94]:
```suggestion

  return 0;
```

Comment (yangc95) [reply content/c/concepts/structures/structures.md:94]: Keeping consistent with block below 🙂

**Review by yangc95 — COMMENTED** [review]
@KTom101 super well written! Gave my two cents which was just some missing/extra lines/spaces but looks great

**Comment by KTom101** [comment]
@yangc95 Thank you!

**Review by Dusch4593 — APPROVED** [review]
Other than @yangc95's comments/suggestions, everything else LGTM 😄

**Comment by KTom101** [comment]
@yangc95 Was there anything else you wanted to add to this and were all your changes met?

**Comment by Dusch4593** [comment]
@yangc95 @KTom101 This seems about ready to merge. I made a small edit in 01d68c1cb3b7762a6fbe705d714da6e4962b7fb9 where I fixed the indentation in the code snippets.

**Pull request merged**